### PR TITLE
Concourse pipeline

### PR DESCRIPTION
**Pull request by akinnane**

### CODE REVIEW

**Review by detnon — APPROVED** [review]
lgtm

File: pipelines/csw-test-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+          #/usr/local/bin/gulp environment.deploy --env=${ENVIRONMENT}
+          deploycsw ${ENVIRONMENT}
+
+    on_success:
+      put: notify
+      params:
+        channel: '#cyber-test-alerts'
+        text: |
+          CSW deploy job has completed.. Check it out at:
+          ((cd-url))/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME
```
Comment (tlwr) [inline pipelines/csw-test-pipeline.yml]:
```suggestion
          $ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME
```

File: pipelines/csw-test-pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+          echo "deploying csw enviroment..."
+          /usr/local/bin/gulp environment.cleanup --env=${ENVIRONMENT}
+
+    on_success:
+      put: notify
+      params:
+        channel: '#cyber-test-alerts'
+        text: |
+          CSW clean-up job has completed. Check it out at:
+          ((cd-url))/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME
```
Comment (tlwr) [inline pipelines/csw-test-pipeline.yml]:
```suggestion
          $ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME
```

File: pipelines/csw-test-pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+          echo "building csw ..."
+          buildcsw 10.11 ${ENVIRONMENT} /root/.ssh/${ENVIRONMENT}.pub
+
+    on_success:
+      put: notify
+      params:
+        channel: '#cyber-test-alerts'
+        text: |
+          CSW build job has completed.. Check it out at:
+          ((cd-url))/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME
```
Comment (tlwr) [inline pipelines/csw-test-pipeline.yml]:
```suggestion
          $ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME
```

File: pipelines/csw-test-pipeline.yml [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+          cd ../build
+          npm install
+
+          echo "loading csw enviroment..."
+          loadcsw 10.11 ${ENVIRONMENT} /root/.ssh/${ENVIRONMENT}.pub
+
+          echo "deploying csw enviroment..."
+          /usr/local/bin/gulp environment.cleanup --env=${ENVIRONMENT}
+
+    on_success:
```
Comment (tlwr) [inline pipelines/csw-test-pipeline.yml:219]: This is repeated multiple times in the job and could be

```
on_success: &notify-slack
...same as current the first time

on_success: *notify-slack # everywhere else - with nothing else
```

**Review by coolteddy — APPROVED** [review]
LGTM

File: pipelines/csw-test-pipeline.yml (hunk 3 above)
Comment (coolteddy) [reply pipelines/csw-test-pipeline.yml]: Thanks @tlwr, done

**Pull request merged**